testapp/settings.py

```diff
@@ -80,3 +80,35 @@
 LOGIN_REDIRECT_URL = reverse_lazy("admin:index")
 
 STATIC_URL = "/static/"
+
+
+# Setup Configuration Settings
```

Comment:
is there any chance all these settings can be bundled into one actual setting with sub-configuration instead? That way we limit the amount of pollution of `django.conf.settings.*`:

Something like:

```py
OIDC_DB_SETUP_CONFIG_ADMIN_AUTH = {
    "OIDC_RP_CLIENT_ID": "client-id",
    "OIDC_RP_SECRET": "secret",
    ...,
}
```

Reply:
Made into a single setting that is a dictionary